app/vmalert/main.go

```diff
@@ -137,6 +140,14 @@ func newManager(ctx context.Context) (*manager, error) {
 		return nil, fmt.Errorf("failed to init remoteRead: %w", err)
 	}
 	manager.rr = rr
+
+	for _, s := range *externalLabels {
+		n := strings.IndexByte(s, '=')
```

Comment:
is better to split by `=` ? 
in that case we can check that only one `=` is present

Reply:
Label values may contain any Unicode characters including `=`, so splitting may complicate the parsing.